tests/platform_tests/api/test_psu.py

```diff
@@ -253,7 +253,11 @@ def test_led(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost, platfo
 
         for psu_id in range(self.num_psus):
             name = psu.get_name(platform_api_conn, psu_id)
-            if name in self.psu_skip_list:
+            if duthost.facts.get("chassis").get("psus")[psu_id].get("led"):
```

Comment:
Can optimize this by calling 
led_support = duthost.facts.get("chassis").get("psus")[psu_id].get("led") 
and then checking for led_support == "N/A"

Reply:
Addressed in the last commit